src/mat/4C_mat_inelastic_defgrad_factors_service.hpp

```diff
@@ -38,8 +40,47 @@ namespace Mat
                                    // failed
   };
 
-  /// map: error types to error messages in InelasticDefgradTransvIsotropElastViscoplast
-  extern std::map<Mat::ViscoplastErrorType, std::string> ViscoplastErrorMessages;
+  /// to_string: error types to error messages in InelasticDefgradTransvIsotropElastViscoplast
+  inline std::string to_string(Mat::ViscoplastErrorType err_type)
+  {
+    switch (err_type)
+    {
+      case Mat::ViscoplastErrorType::NegativePlasticStrain:
+        return "Error in InelasticDefgradTransvIsotropElastViscoplast: negative plastic strain!";
+        break;
```

Comment:
The `break`s are irrelevant here because you anyways return before. Furthermore, the formatting (removing obsolete linebreaks) could be enhanced a bit.

Reply:
Yes, thank you! I will enhance this in a new commit, along with your comment above